web/src/search.js

```diff
@@ -6,12 +6,14 @@ import {Filter} from "./filter";
 import * as keydown_util from "./keydown_util";
 import * as message_view_header from "./message_view_header";
 import * as narrow_state from "./narrow_state";
+import * as search_pill from "./search_pill";
 import * as search_suggestion from "./search_suggestion";
 
 // Exported for unit testing
 export let is_using_input_method = false;
+export let search_pill_widget = null;
 
-export function narrow_or_search_for_term(search_string, {on_narrow_search}) {
+function narrow_or_search_for_term({on_narrow_search}) {
```

Comment:
The `export` already wasn't needed, I can make a separate commit for removing it.